### PR TITLE
Rst 1092

### PR DESCRIPTION
**Pull request by sivvoy**

### CODE REVIEW

File: features/completed-form-available-to-download.feature [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: One other person making claim
+    Given I am making a claim with one other person
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: Validate TXT file when uploading CSV data with 7 other people making the claim
+    Given I am making a claim with 7 other people
```
Comment (garytaylor) [inline features/completed-form-available-to-download.feature]: You can make a claim with multiple people using either the web form (limited number) or using csv. Maybe it is worth adding some wording in this setup step to suggest that you are using csv here ?

File: features/completed-form-available-to-download.feature [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  Scenario: Uploading RTF data when describing your claim in details
+    Given I am making a claim using a separate document
+    When I submit a completed Employment Tribunal form
+    Then the completed form in RTF format is available for ATOS to download and validate
+
+  Scenario: Making claim against 3 employers
+    Given I am making a claim against 3 employers
+    When I submit a completed Employment Tribunal form
+    Then the completed form in TXT format should contained 3 employers details
```
Comment (garytaylor) [inline features/completed-form-available-to-download.feature]: In the other scenarios, we are using ' ... is available for ATOS to download' - i.e. we are saying that we are examining the file from the atos interface.  But, in this one it just talks about 'the completed form in txt format', not stating that you are looking at the file from the atos interface, but from 'somewhere unknown' !

Also, I think (but could be wrong !) the step such as 'Then the uploaded CSV data is formatted and available for ATOS to download' checks the contents of the file you are talking about based on the given input data - so is there any need to do a step that says 'should contain 3 employers details' ?

I must admit, it is more explicit though which is nice - just inconsistent with the others.

File: features/factories/claimant_factory.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      locality 'London'
+      county 'Greater London'
+      post_code 'SW1H 9AJ'
+    end
+
+    trait :group_claims do
+      group_claims_csv 'simple_user_with_csv_group_claims.csv'
+    end
+
+    factory :first_person, traits: [:answer_data, :person_data]
```
Comment (garytaylor) [inline features/factories/claimant_factory.rb:34]: Ive seen this in the examples in factory bot, but never really understood them.  We kind if have nested factories - i.e. :first_person within :claimant - how is this used ?

File: features/completed-form-available-to-download.feature [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+@javascript
+Feature:
+  As an ATOS admin
+  I want to be able to download completed forms
+  So I can triage an employees claim against their employer
+
+  Scenario: PDF format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
```
Comment (garytaylor) [inline features/completed-form-available-to-download.feature]: thinking ahead a little - as we are gonna be talking about a claim form and a respondent form once et3 stuff is included - might be worth just calling everything a claim form now ?

**Review by garytaylor — APPROVED** [review]
A few comments - but all looks good :-) bye bye personas.yml

File: features/completed-form-available-to-download.feature (hunk 1 above)
Comment (sivvoy) [reply features/completed-form-available-to-download.feature]: Is this OK? 

 Scenario: Validate TXT file when uploading CSV data
    Given 7 employees making a claim by uploading CSV file
    When the completed form is submitted
    Then I can download the uploaded CSV data in TXT format

Comment (garytaylor) [reply features/completed-form-available-to-download.feature]: Yeah that reads better I think

File: features/completed-form-available-to-download.feature (hunk 2 above)
Comment (sivvoy) [reply features/completed-form-available-to-download.feature]: Test is to check check that the txt file contained 3 employers details

Scenario: Making claim against 3 employers
    Given an employee making a claim against 3 employers
    When the completed form is submitted
    Then I can download the form and validate the TXT file contained 3 employers details

File: features/factories/claimant_factory.rb (hunk 3 above)
Comment (sivvoy) [reply features/factories/claimant_factory.rb:34]: I just used this to create a list e.g @claimants = FactoryBot.create_list(:first_person, 2, :person_data)

Comment (garytaylor) [reply features/factories/claimant_factory.rb:34]: right gotcha :-)

**Pull request merged**